<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Patients Data</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="drugs">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="patient_tab">
       <attribute name="title">
        <string>Patients</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QToolBox" name="toolBox">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="add_remove_page">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>636</width>
             <height>374</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Add / Remove </string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="2" column="4">
             <widget class="QPushButton" name="pushButton_3">
              <property name="text">
               <string>Clear</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" rowspan="2" colspan="2">
             <widget class="QListWidget" name="listWidget"/>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="pushButton_4">
              <property name="text">
               <string>Remove First</string>
              </property>
             </widget>
            </item>
            <item row="0" column="4" rowspan="2">
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLineEdit" name="id">
                <property name="statusTip">
                 <string notr="true"/>
                </property>
                <property name="echoMode">
                 <enum>QLineEdit::Normal</enum>
                </property>
                <property name="clearButtonEnabled">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="name">
                <property name="clearButtonEnabled">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="gender">
                <item>
                 <property name="text">
                  <string>Male</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Female</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="phone">
                <property name="clearButtonEnabled">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="address">
                <property name="clearButtonEnabled">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="dob">
                <property name="inputMethodHints">
                 <set>Qt::ImhNone</set>
                </property>
                <property name="clearButtonEnabled">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="history">
                <property name="clearButtonEnabled">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="allergy">
                <property name="clearButtonEnabled">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="syst">
                <property name="maximum">
                 <number>300</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="diast">
                <property name="maximum">
                 <number>300</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="heart">
                <property name="maximum">
                 <number>300</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="temp">
                <property name="maximum">
                 <double>301.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="msgLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="textFormat">
               <enum>Qt::AutoText</enum>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="3" rowspan="2">
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="idLabel">
                <property name="text">
                 <string>Patient ID</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="nameLabel">
                <property name="text">
                 <string>Name</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="genderLabel">
                <property name="text">
                 <string>Gender</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="phoneLabel">
                <property name="text">
                 <string>Phone</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="addressLabel">
                <property name="text">
                 <string>Address</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="dobLabel">
                <property name="text">
                 <string>Birth Date</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="historyLabel">
                <property name="text">
                 <string>Med History</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="allergyLabel">
                <property name="text">
                 <string>Allergy</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="systLabel">
                <property name="text">
                 <string>Systolic</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="diastLabel">
                <property name="text">
                 <string>Diastolic</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="heartLabel">
                <property name="text">
                 <string>Heart Rate</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="tempLabel">
                <property name="text">
                 <string>Temperature</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Rear ptr</string>
                </property>
                <property name="textFormat">
                 <enum>Qt::AutoText</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLCDNumber" name="lcdNumber_rear"/>
              </item>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Front ptr</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLCDNumber" name="lcdNumber_front"/>
              </item>
             </layout>
            </item>
            <item row="0" column="0">
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>ICU Number</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_4">
               <item row="1" column="0">
                <widget class="QRadioButton" name="radioButton_2">
                 <property name="text">
                  <string>ICU 2</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QRadioButton" name="radioButton_3">
                 <property name="text">
                  <string>ICU 3</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QRadioButton" name="radioButton">
                 <property name="text">
                  <string>ICU 1</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QPushButton" name="pushButton_2">
              <property name="text">
               <string>Insert</string>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>Close</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="drugs_page">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>653</width>
             <height>356</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Drugs</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="4" column="5">
             <widget class="QLabel" name="every_2">
              <property name="text">
               <string>Every</string>
              </property>
             </widget>
            </item>
            <item row="4" column="7">
             <widget class="QLabel" name="label_8">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>hr</string>
              </property>
             </widget>
            </item>
            <item row="5" column="3">
             <widget class="QLabel" name="time3LAbel">
              <property name="text">
               <string>Time</string>
              </property>
             </widget>
            </item>
            <item row="4" column="6">
             <widget class="QSpinBox" name="spinBox_every2"/>
            </item>
            <item row="0" column="1" colspan="7">
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="0">
               <widget class="QListWidget" name="listWidget_drugs">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QTreeWidget" name="treeWidget_drugs">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <column>
                 <property name="text">
                  <string>Patient</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Drug</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Time</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Every (hr)</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Date</string>
                 </property>
                </column>
               </widget>
              </item>
             </layout>
            </item>
            <item row="3" column="7">
             <widget class="QLabel" name="label_7">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>hr</string>
              </property>
             </widget>
            </item>
            <item row="5" column="7">
             <widget class="QLabel" name="label_12">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>hr</string>
              </property>
             </widget>
            </item>
            <item row="2" column="4" colspan="2">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Leave Drug empty if not changed</string>
              </property>
             </widget>
            </item>
            <item row="4" column="4">
             <widget class="QTimeEdit" name="time2_drugs"/>
            </item>
            <item row="5" column="1">
             <widget class="QLabel" name="drug_label_3">
              <property name="text">
               <string>Drug 3</string>
              </property>
             </widget>
            </item>
            <item row="5" column="2">
             <widget class="QLineEdit" name="drug_3">
              <property name="clearButtonEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="4">
             <widget class="QTimeEdit" name="time1_drugs"/>
            </item>
            <item row="3" column="0">
             <widget class="QPushButton" name="add_drugs">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
            <item row="4" column="3">
             <widget class="QLabel" name="time2LAbel">
              <property name="text">
               <string>Time</string>
              </property>
             </widget>
            </item>
            <item row="4" column="2">
             <widget class="QLineEdit" name="drug_2">
              <property name="clearButtonEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QPushButton" name="edit_drugs">
              <property name="text">
               <string>edit today</string>
              </property>
             </widget>
            </item>
            <item row="3" column="5">
             <widget class="QLabel" name="every_1">
              <property name="text">
               <string>Every</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLabel" name="drug_label_1">
              <property name="text">
               <string>Drug 1</string>
              </property>
             </widget>
            </item>
            <item row="5" column="6">
             <widget class="QSpinBox" name="spinBox_every3"/>
            </item>
            <item row="2" column="2">
             <widget class="QLineEdit" name="id_drugs">
              <property name="clearButtonEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLabel" name="drug_label_2">
              <property name="text">
               <string>Drug 2</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QGroupBox" name="groupBox_2">
              <property name="title">
               <string>ICU Number</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_7">
               <item row="1" column="0">
                <widget class="QRadioButton" name="radioButton_icu2_drugs">
                 <property name="text">
                  <string>ICU 2</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QRadioButton" name="radioButton_icu3_drugs">
                 <property name="text">
                  <string>ICU 3</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QRadioButton" name="radioButton_icu1_drugs">
                 <property name="text">
                  <string>ICU 1</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QLineEdit" name="drug_1">
              <property name="clearButtonEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="5" column="5">
             <widget class="QLabel" name="every_3">
              <property name="text">
               <string>Every</string>
              </property>
             </widget>
            </item>
            <item row="5" column="4">
             <widget class="QTimeEdit" name="time3_drugs"/>
            </item>
            <item row="3" column="6">
             <widget class="QSpinBox" name="spinBox_every1"/>
            </item>
            <item row="1" column="1" colspan="7">
             <widget class="Line" name="line">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="patientidLabel_drug">
              <property name="text">
               <string>Patient ID</string>
              </property>
             </widget>
            </item>
            <item row="3" column="3">
             <widget class="QLabel" name="time1LAbel">
              <property name="text">
               <string>Time</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QPushButton" name="close_drugs">
              <property name="text">
               <string>Close</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>695</width>
     <height>21</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>dob</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>646</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>645</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>address</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>646</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>645</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>syst</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>646</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>645</x>
     <y>325</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>id</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>646</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>645</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>allergy</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>646</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>645</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>heart</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>646</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>645</x>
     <y>377</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>temp</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>646</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>645</x>
     <y>403</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>name</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>646</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>645</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>phone</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>646</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>645</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>history</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>646</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>645</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>diast</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>646</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>645</x>
     <y>351</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>lcdNumber_front</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>403</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>id</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButton_2</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>465</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>433</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>lcdNumber_rear</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>448</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>410</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>close_drugs</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>453</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>464</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
